plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java

```diff
@@ -307,6 +307,27 @@ public void testRegisterTableWithMetadataFile()
         assertUpdate(format("DROP TABLE %s", tableName));
     }
 
+    @Test
+    public void testRegisterTableWithTrailingSpaceInLocation()
```

Comment:
I think it would be better to split into two test methods, create table & register table. The "create table" should not be overridden and "register table" should be overridden only by REST test.